src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -65,6 +65,14 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
         OPPORTUNITIES
     }
 
+    /** @description Limit for query locator. Property is public to allow for testing */
+    public Integer chunkSize = 10000000;
+
+    /** @description Last Id processed by the batch so query locator can return records in ID
+     *  order and batch can be chained
+    */
+    public Id lastProcessedId;
```

Comment:
@seanpat09 , do you intend to refactor PK chunking in batches in this WI?

Reply:
After looking at the work that John Brandolini is doing and from the feedback from my proposal, it seems that there are more considerations/features needed before we do a refactor. With this WI we can at least address the bug more quickly